package.json

```diff
@@ -127,7 +127,7 @@
     "pg": "^7.3.0",
     "plop": "^1.5.0",
     "prettier": "^1.6.1",
-    "puppeteer": "^0.10.2",
+    "puppeteer": "^1.13.0",
```

Comment:
There was a vulnerability with `puppeteer` that required upgrading it. Although this most likely can't cause any issues, upgrading it was easy.